packages/components/button/src/use-button.ts

```diff
@@ -112,6 +113,7 @@ export function useButton(props: UseButtonProps) {
         fullWidth,
         isDisabled,
         isInGroup,
+        isIsolate,
```

Comment:
Should this not be `isIsolated` instead of `isIsolate`?
```suggestion
        isIsolated,
```

Reply:
Looks better!

Reply:
It's been removed the ```isIsolated``` condition.